packages/vscode-extension/.vscode/launch.json

```diff
@@ -7,7 +7,7 @@
   "configurations": [
     {
       "name": "Run Extension",
-      "type": "pwa-extensionHost",
```

Comment:
what's the difference between pwa-ext vs. ext?

Reply:
+1, why modify this?

Reply:
latest vscode does not support "pwa-extensionHost"